codec-http2/src/main/java/io/netty/handler/codec/http2/hpack/Decoder.java

```diff
@@ -0,0 +1,568 @@
+/*
+ * Copyright 2015 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
```

Comment:
https://github.com/netty/netty/blob/master/LICENSE.txt#L124:

> You may add Your own copyright statement to Your modifications

Are we saying that adding an additional copyright is not valid, or just that we want/need to have the special `The Netty Project licenses this file to you` prefix?  Why do we need this prefix at all?